source/isaaclab/isaaclab/envs/manager_based_rl_env.py

```diff
@@ -82,6 +82,9 @@ def __init__(self, cfg: ManagerBasedRLEnvCfg, render_mode: str | None = None, **
 
         # initialize data and constants
         # -- init buffers
+        # TODO: this may be redundant since self.episode_length_buf is now initialized in load_managers() to make it
```

Comment:
Instead of moving it to load_managers (which is only for managers and not buffers), maybe it is better to move the buffer initialization further up in the class